app/portainer/services/api/userService.js

```diff
@@ -157,6 +136,40 @@ angular.module('portainer.app').factory('UserService', [
       return deferred.promise;
     };
 
+    service.createAccessToken = function (id, description) {
+      const deferred = $q.defer();
```

Comment:
can we use regular async/await instead of `$q`?

Reply:
I didnt write this code (but moved it).
I did approved this because all the other code in this file is using `$q`. I think its better for this code to be similar to the other code - as all of it would probably be migrated to async/await/react at the same time.